packages/wrangler/src/deployment-bundle/bundle.ts

```diff
@@ -367,7 +391,8 @@ export async function bundleWorker(
 		sourceRoot: destination,
 		minify,
 		metafile: true,
-		conditions: BUILD_CONDITIONS,
+		conditions: getBuildConditions(),
+		platform: getBuildPlatform(),
```

Comment:
how is `platform` passed further down to esbuild? Searching through the codebase, it would seem that `platform` is always hard-coded to mostly "node"

Reply:
This is where we hand it to esbuild. Is "platform" used elsewhere? Where is it hard-coded to `node`? Did I miss something?

Reply:
lol...I totes missed this was in `bundleWorker`